resources/osl_heisenbridge.rb

```diff
@@ -0,0 +1,38 @@
+# osl_heisenbridge
+#
+# Heisenbridge is an IRC bouncer for a Matrix server
+
+resource_name :osl_heisenbridge
+provides :osl_heisenbridge
+unified_mode true
+
+default_action :create
+
+property :container_name, String, name_property: true
```

Comment:
Not sure this is the best name based on what I see below.

Reply:
In order to make linking App Services, like Heisenbridge, to the main Synapse server easy for the developer. It's best if everything related to the container is named the exact same.

I have experimented with instead updating the App Service list using attributes inside of the resources, but removed it as it goes against the spirit of resources. (See commented out code in osl_heisenbridge)